main.css

```diff
@@ -0,0 +1,621 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,900;1,300&family=Poppins:wght@400;600;700;900&display=swap');
+
+html,
+body {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+
+  --primary-color: #272a31;
+  --secondary-color: #d3d3d3;
+  --accent-color: #ec4542;
+  --shadow: 0 2px 10px -1px rgba(0, 0, 0, 0.3);
+
```

Comment:
Please ensure that the pages use the exact same colors, for text, backgrounds, borders, etc, from the original design.

* Basic color swatch: #272A31, #EC5242, #D3D3D3

Also, please ensure that all page elements have similar spacing, borders, shadows, and paddings as in the original design.